### PR TITLE
Refactor: insert new class refactoring

### PR DESCRIPTION
**Pull request by balsa-sarenac**
`RBInserClassTransformation` and `RBInsertClassRefactoring` have the same duplicate logic. This PR addresses this issue by:
* migrating usages of `RBInserClassTransformation` to refactoring
* tagging transformation as unused
* renaming the refactoring so it's more clear that it will produce a new class

### CODE REVIEW

**Comment by balsa-sarenac** [comment]
The build failed, but I think it's unrelated issue.

**Comment by jecisc** [comment]
Indeed, the issue is unrelated

File: src/Refactoring2-Transformations/RBInsertNewClassTransformation.class.st [hunk 1]
```diff
@@ -1,35 +1,16 @@
...
-	category: #'Refactoring2-Transformations-Tests')
-	transform.
-(ChangesBrowser changes: transformation model changes changes) open
-
-Preconditions:
-- superclass must be an existing class
-- each subclass also must be an existing class
-- name of the class to be added must be non-existent in the system
-- name of the category must be a valid one
-"
```
Comment (jecisc) [inline src/Refactoring2-Transformations/RBInsertNewClassTransformation.class.st]: There is a bug currently in Pharo 12 that makes some comments vanish. And it happened here.

(A fix for this bug was just integrated)

**Review by jecisc — CHANGES_REQUESTED** [review]

File: src/Refactoring2-Transformations/RBInsertNewClassTransformation.class.st (hunk 1 above)
Comment (balsa-sarenac) [reply src/Refactoring2-Transformations/RBInsertNewClassTransformation.class.st]: Nice catch @jecisc! I've just returned the removed comment. Glad that was fixed, it was a nasty bug.

**Pull request merged**